providers/src/airflow/providers/dbt/cloud/utils/openlineage.py

```diff
@@ -156,6 +156,8 @@ async def get_artifacts_for_steps(steps, artifacts):
             task_id=operator.task_id,
             logical_date=_get_logical_date(task_instance),
             try_number=_get_try_number(task_instance),
+            queued_dttm=task_instance.queued_dttm,
```

Comment:
queued datetime can be updated. this would not be deterministic.

Reply:
what's the thinking behind using queued dttm?

Reply:
In this case, I mean two things: as would not change _during_ run on the worker - and will change when the task gets rescheduled. I've seen that to be true - do you think it's not?

As for the thinking, it was the only TI param I've seen changing with mode=reschedule sensors - I need to distinguish between different runs of those.

Reply:
We recently added a feature, when a task gets stuck in queued, then we may set it back to scheduled and queue it again.  The queued time is not part of what defines a TI so i don't see a reason to include it here.  Did you have any particular reason that you thought it was _needed_?  (Beyond a reason you think it _wouldn't hurt_)

Reply:
Bit confused.

> The issue with reschedule is getting duplicate calls to listener, with all parameters the same, besides queued_datetime. This causes duplicate OL start events with same run_id, which is against OL spec

Can you expand on this a bit.  what do you mean "the issue with reschedule"? i'm not sure which issue or scenario you're referring to.

Reply:
Oh you are talking about sensors in reschedule mode.  Why do you want to create a new ID for each time the task "reawakens"?  Isn't it all part of the same "run"?

Reply:
If you use mode=reschedule, the [on_task_instance_running](https://airflow.apache.org/docs/apache-airflow/stable/administration-and-deployment/listeners.html#taskinstance-state-change-events) listener will be called multiple times with the same task instance. 

There is apparently any other way to distinguish those events other than `queued_dttm` which is getting updated when scheduler schedules it again. Compare to the retry scenario, where `try_number` is being bumped for any additional retry.

Reply:
So will it appear like a new run each time it is called?  Perhaps it should not be called each time?

Reply:
We have no option to determine whether the reschedule predicate will succeed, besides actually executing it 😶

Reply:
I just mean call on_task_instance_running the first time but not in subsequent times.  Or essentially, like, let the duplicate signals get ignored.  What's the point of having say 10 OL start events for the same TI?  it's sort of an implementation detail that we suspend and resume sensors with reschedule mode.  nothing new is happening each time.  it's still in effect one long continuous running phase, just with a suspend-resume sort of sub-phase, you know?

Reply:
@dstandish The issue still is that we don't know whether it was called before or not. There is no information about that. If you think there's something that would allow us to know that, then I'll be happy to try it out.

Reply:
I think if queued time after start time, might work. But, I scheduled a zoom